content/en/docs/components/pipelines/overview/multi-user.md

```diff
@@ -1,175 +1,179 @@
+This method stores your user namespace in a configuration file at `$HOME/.config/kfp/context.json`.
+{{% /alert %}}
 
-Detailed documentation for the Kubeflow Pipelines SDK can be found in the
-[Kubeflow Pipelines SDK Reference](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.client.html).
+{{% alert title="Tip" color="info" %}}
+Detailed documentation for `kfp.Client()` can be found in the [Kubeflow Pipelines SDK Reference](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.client.html).
+{{% /alert %}}
 
-### When using REST API or generated python API client
+## When using the REST API
 
-Similarly, when calling [REST API endpoints](/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/)
-or using [the generated python API client](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html),
-namespace argument is required for experiment APIs. Note that namespace is
-referred to using a resource reference. The resource reference **type** is
-`NAMESPACE` and resource reference **key id** is the namespace name.
+When calling the [Kubeflow Pipelines REST API](/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/), a namespace argument is required for experiment APIs. 
 
-The following example demonstrates how to use [the generated Python API client (kf-server-api)](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html) in a multi-user environment.
+{{% alert title="Tip" color="info" %}}
+The namespace is specified by a "resource reference" with **type** of `NAMESPACE` and **key.id** equal to the namespace name.
+{{% /alert %}}
+
+The following code uses the [generated python API client](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html) to create an experiment and pipeline run:
 
 ```python
-from kfp_server_api import ApiRun, ApiPipelineSpec, \
-    ApiExperiment, ApiResourceType, ApiRelationship, \
-    ApiResourceReference, ApiResourceKey
-# or you can also do the following instead
-# from kfp_server_api import *
-
-experiment=client.experiments.create_experiment(body=ApiExperiment(
-    name='test-experiment-1234',
-    resource_references=[ApiResourceReference(
-        key=ApiResourceKey(
-            id='<namespace>', # Replace with your own namespace.
-            type=ApiResourceType.NAMESPACE,
-        ),
-        relationship=ApiRelationship.OWNER,
-    )],
-))
+import kfp
+from kfp_server_api import *
+
+# the namespace in which you deployed Kubeflow Pipelines
+kubeflow_namespace = "kubeflow"
+
+# the namespace of your pipelines user (where the pipeline will be executed)
+user_namespace = "jane-doe"
+
+# when `path` is None, the `KF_PIPELINES_SA_TOKEN_PATH` environment variable is used
+# (if `KF_PIPELINES_SA_TOKEN_PATH` is unset, the default is: `/var/run/secrets/kubeflow/pipelines/token`)
+credentials = kfp.auth.ServiceAccountTokenVolumeCredentials(path=None)
```

Comment:
Is this step necessary? Or we are listing here in case people want to use it?

Reply:
It's only explicitly necessary if the token is not mounted at the default location, but since we have no idea how the user's environment looks, I think it's best to be explicit.